Library/Formula/ufraw.rb

```diff
@@ -1,9 +1,8 @@
-require 'formula'
-
 class Ufraw < Formula
-  homepage 'http://ufraw.sourceforge.net'
-  url 'https://downloads.sourceforge.net/project/ufraw/ufraw/ufraw-0.20/ufraw-0.20.tar.gz'
-  sha1 'f2f456c6ec5ab128433502eae05b82a7ed636f3e'
+  homepage "http://ufraw.sourceforge.net"
+  desc "RAW image processing utility"
```

Comment:
Actually, I can fix this when I pull. No point queuing another build on the bot when it's somewhat busy tonight.